src/components/hooks/useStream.js

```diff
@@ -10,12 +10,21 @@ export default function useStream(stream) {
   const streamRef = useRef({});
 
   useEffect(() => {
-    const mediaElement = streamRef.current;
+    const playStream = async () => {
+      try {
+        const mediaElement = streamRef.current;
 
-    if (stream instanceof MediaStream) {
-      mediaElement.srcObject = stream;
-      mediaElement.play();
-    }
+        if (stream instanceof MediaStream) {
+          mediaElement.srcObject = stream;
+          await mediaElement.play();
+        }
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error('new stream is being added. not sure if we should error this out');
```

Comment:
Perhaps a more informative error message

```suggestion
        console.error('Unable to play stream: ', error.message);
```

Reply:
method is removed